src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -98,8 +98,15 @@ ReportCmcStatus::writeReport( const xrt_core::device* /*_pDevice*/,
   try {
     boost::property_tree::ptree cmc_scale = cmc.get_child("scaling");
     _output << boost::format("  %-22s :\n") % "Runtime clock scaling feature";
-    _output << boost::format("    %s : %s\n") % "Supported" % cmc_scale.get<std::string>("supported");
-    _output << boost::format("    %s : %s\n") % "Enabled" % cmc_scale.get<std::string>("enabled");
+    if (!cmc_scale.get<bool>("supported")) {
+      _output << "    Information unavailable or not applicable" << std::endl;

```

Comment:
```suggestion
      _output << "    Information unavailable\n";

```
To match the other messages. Most other reports just state "Information Unavailable"